integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -233,5 +352,33 @@ fn test_resharding_v3_drop_chunks_all() {
         (ShardUId { shard_id: 2, version: 3 }, 0..3),
         (ShardUId { shard_id: 3, version: 3 }, 0..1),
     ]);
-    test_resharding_v3_base(chunk_ranges_to_drop);
+    test_resharding_v3_base(
+        TestReshardingParameters::default().chunk_ranges_to_drop(chunk_ranges_to_drop),
+    );
+}
+
+#[test]
+#[ignore]
+#[cfg(feature = "test_features")]
+fn test_resharding_v3_resharding_block_in_fork() {
+    let mut params = TestReshardingParameters::default();
+    let client = params.clients[0].clone();
+    params = params
+        .clients(vec![client.clone()])
+        .block_and_chunk_producers(vec![client])
+        .loop_action(Some(fork_before_resharding_block(false)));
```

Comment:
Maybe make a `TestReshardingParametersBuilder` to avoid setting parameters twice? Or implement this using constructor `_with_clients(num_clients)` where default is 3 and we select clients with indices `i * 3`.

Reply:
I went with the `with_client` constructor.